src/jit/flowgraph.cpp

```diff
@@ -2407,33 +2407,70 @@ BlockSet_ValRet_T   Compiler::fgDomFindStartNodes()
     return startNodes;
 }
 
-/** A simple DFS traversal of the flow graph.
-  * It computes both preorder and postorder numbering.
+/** A non-recursive DFS traversal of the flow graph using an
+  * evaluation stack to assign post-order numbers.
   */
 void Compiler::fgDfsInvPostOrderHelper(BasicBlock* block, BlockSet& visited, unsigned* count)
```

Comment:
We've never gone through and made existing code conform to CC. Maybe we should to avoid confusion especially with new people in the source base. Look at conventions.txt in the JIT directory as a shortcut, or see https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md.